app/src/main/res/drawable/ic_directions_washroom.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we use a different washroom icon? like a toilet or something more descriptive?

Reply:
I couldn't find any :/

Reply:
How about wc ? (through the android clipart thing)

![image](https://user-images.githubusercontent.com/32105523/79279649-7d36a880-7e7c-11ea-85aa-f3c5e70745aa.png)

Reply:
Ohhhh didn't see this yeah I'll replace it !